metashare/api/models.py

```diff
@@ -168,6 +168,10 @@ class Repository(
     description = MarkdownField(blank=True, property_suffix="_markdown")
     is_managed = models.BooleanField(default=False)
     repo_id = models.IntegerField(null=True, blank=True, unique=True)
+    commits = JSONField(default=list)
+    branch_name = models.CharField(
+        max_length=100, blank=True, null=True, validators=[validate_unicode_branch]
+    )
```

Comment:
Can we prepopulate this from the repo `default_branch`?

Reply:
Yes.

Reply:
No, because that no longer exists! Marking resolved.